pom.xml

```diff
@@ -169,6 +169,7 @@
     <junit.version>4.13</junit.version>
     <junit.jupiter.version>5.6.2</junit.jupiter.version>
     <junit.vintage.version>5.6.2</junit.vintage.version>
+    <kafka.version>2.5.0</kafka.version>
```

Comment:
Kafka version is also declared specifically in kafka-handler/pom.xml Can we parmeterize there, so that all of Hive is referencing just one kafka version?

Reply:
sure!